RZBluetooth/Profiles/RZBDeviceInfo.m

```diff
@@ -35,17 +144,51 @@ + (CBCharacteristicProperties)characteristicPropertiesForKey:(NSString *)key
 
 + (id)valueForKey:(NSString *)key fromData:(NSData *)data
 {
-    return [[NSString alloc] initWithData:data encoding:NSUTF8StringEncoding];
+    if ([key isEqualToString:@"systemId"]) {
+        return [RZBSystemId valueForKey:key fromData:data];
+    }
+    else
+    if ([key isEqualToString:@"pnpId"]) {
```

Comment:
👍 No problem. Personal preference. 

I'll update the PR with these changes as soon as I can get back to it, probably tomorrow.